core/src/test/java/com/yahoo/oak/ComputeTest.java

```diff
@@ -116,14 +127,18 @@ public void testMain() throws InterruptedException {
             oak.zc().putIfAbsent(key, val);
         }
 
-        for (int i = 0; i < NUM_THREADS; i++) {
-            threads.get(i).start();
-        }
-
         latch.countDown();
 
-        for (int i = 0; i < NUM_THREADS; i++) {
-            threads.get(i).join();
+        executor.shutdown();
+        try {
+            executor.shutdown();
```

Comment:
Why do we use `shutdown()` twice? I guess it is a mistake, right?

Reply:
yes it is a mistake